Error on create-sim

I followed all instructions but i'm unable to create a new sim.

This is the error that i'm getting:

`atal error: Perennial task failed:
Error: grunt unpublished-README in ../MMGD failed with exit code 3 and stdout:
Loading "Gruntfile.js" tasks...ERROR
>> Error: Cannot find module 'lodash'
Warning: Task "unpublished-README" not found. Use --force to continue.

Aborted due to warnings.

  at ChildProcess.process.on.code (/home/rahto/Documents/phet/phetsims/perennial/js/common/execute.js:72:17)
  at emitTwo (events.js:126:13)
  at ChildProcess.emit (events.js:214:7)
  at maybeClose (internal/child_process.js:925:16)
  at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)

Full Error details:
{
  "cmd": "grunt",
  "args": [
    "unpublished-README"
  ],
  "cwd": "../MMGD",
  "stdout": "Loading \"Gruntfile.js\" tasks...ERROR\n>> Error: Cannot find module 'lodash'\nWarning: Task \"unpublished-README\" not found.\u0007 Use --force to continue.\n\nAborted due to warnings.\n",
  "stderr": "",
  "code": 3
}
`

--- a/doc/phet-development-overview.md
+++ b/doc/phet-development-overview.md
@@ -189,14 +189,17 @@ After checking out the dependencies and installing grunt-cli in the preceding in
 1. Check out the template sim, called ‘simula-rasa’ using this git clone command:
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`cd phetsims`
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`git clone https://github.com/phetsims/simula-rasa.git`
-2. Install the perennial dependencies:
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`cd perennial`
+2. Install the chipper dependencies:
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`cd chipper`
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`npm install`
-3. Use the perennial ‘grunt’ task to create a new sim, like so (still in the perennial directory):
+3. Install the perennial dependencies:
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`cd ../perennial`
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`npm install`
+4. Use the perennial ‘grunt’ task to create a new sim, like so (still in the perennial directory):
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`grunt create-sim --repo=NAME --author=AUTHOR`
 For instance, if the simulation is going to be named Acceleration Lab and the author is Jane Doe, then you could put:
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`grunt create-sim --repo=acceleration-lab --author="Jane Doe"`
-4. Test the created simulation in the browser and make sure it launches.  It should be a blank simulation.  Write to the Developing Interactive Simulations in HTML5 Google Group if you run into problems.
+5. Test the created simulation in the browser and make sure it launches.  It should be a blank simulation.  Write to the Developing Interactive Simulations in HTML5 Google Group if you run into problems.
 
 ## Building and Testing (Windows and macOS)
 

